spec/Spec/BlsEC.cry

```diff
@@ -28,6 +28,15 @@ type G1Point =
     , yCoord : Bytes48
     }
 
+/**
+ * Representation of G1 point at infinity.
+ */
```

Comment:
suggestion: You could consider explaining that `(0,0)` isn't actually the point at infinity, but since we can't use enums here and it's not on the curve, it's an acceptable substitute. But I think this project isn't really about the EC math as much as it's about the applications, so up to you.

Reply:
This is an excellent suggestion. I'll add it --- thanks!